org/all-prs.js

```diff
@@ -51,6 +52,9 @@ const unknownPrefix = compose(none(Boolean), ap(map(startsWithI, commitTypes)),
 const commitsWithUnkownType =
   filter(compose(unknownPrefix, commitMsg));
 
+const excludeMergeCommits =
+  filter(compose(complement(startsWith('Merge')), commitMsg));
```

Comment:
`filter(complement(f))` can be written `reject(f)`.

Reply:
Done!